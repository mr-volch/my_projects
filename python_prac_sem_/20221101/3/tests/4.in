a = Grange(10,10,1000000000000000000000000000000)
print(a)
print(tuple(a))
print(len(a))
print(a[0], a[22])
b = a[100:10000]
c = a[100:10000:100]
d = Grange(10,1,2)
print(list(b), list(c), list(d))
print(b,c,d)
print(not(d), not(b))
print(len(b), len(c))
