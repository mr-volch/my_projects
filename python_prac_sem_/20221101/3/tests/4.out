grange(10, 10, 1000000000000000000000000000000)
(10, 100, 1000, 10000, 100000, 1000000, 10000000, 100000000, 1000000000, 10000000000, 100000000000, 1000000000000, 10000000000000, 100000000000000, 1000000000000000, 10000000000000000, 100000000000000000, 1000000000000000000, 10000000000000000000, 100000000000000000000, 1000000000000000000000, 10000000000000000000000, 100000000000000000000000, 1000000000000000000000000, 10000000000000000000000000, 100000000000000000000000000, 1000000000000000000000000000, 10000000000000000000000000000, 100000000000000000000000000000)
29
10 100000000000000000000000
[100, 1000] [100] []
grange(100, 10, 10000) grange(100, 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, 10000) grange(10, 1, 2)
True False
2 1
